app/src/main/java/com/example/journeyjournal/ParseConnectorFiles/Post.java

```diff
@@ -46,14 +47,12 @@ public void setUser(ParseUser user) {
         put(KEY_USER, user);
     }
 
-    public ParseGeoPoint getLocation() {
-        return getParseGeoPoint(KEY_LOCATION);
-    }
-
     public void setLocation(ParseGeoPoint location) {
```

Comment:
remove this also

Reply:
Removed
<img width="908" alt="Screen Shot 2022-07-22 at 11 51 35 AM" src="https://user-images.githubusercontent.com/93938197/180504839-ee00e738-0a07-4c91-9f76-1154e21bfd79.png">